src/tests/test_dynamic_api/test_views_wfs.py

```diff
@@ -58,6 +58,26 @@ def test_wfs_field_public(
             "straatnaam": None,
         }
 
+    def test_wfs_model_auth(
+        self, api_client, parkeervakken_schema, parkeervakken_parkeervak_model
```

Comment:
goed om te weten: als je `parkeervakken_parkeervak_model` binnenhaalt, zit `parkeervakken_schema` daar al bij. Deze kan je dus weglaten (maar dat zit er nog bij meer tests in)

Reply:
Ok, voor de grote opschoonronde :)